diff -aurN trunk.orig/src/gx_head/wscript trunk/src/gx_head/wscript
--- trunk.orig/src/gx_head/wscript	2012-07-03 23:03:04.628627703 +0200
+++ trunk/src/gx_head/wscript	2012-07-03 23:22:36.255980344 +0200
@@ -75,7 +75,7 @@
     if sys.platform.startswith("linux"):
         lib.append('dl')
     uselib = ['JACK', 'SNDFILE', 'GTHREAD', 'GMODULE_EXPORT',
-              'GTK2', 'GTKMM', 'GIOMM', 'FFTW3', 'LRDF']
+              'GTK2', 'GTKMM', 'GIOMM', 'FFTW3', 'LRDF', 'BOOST_SYSTEM']
     uselib_local = []
     if bld.env['ZITA_CONVOLVER']:
         uselib.append('ZITA_CONVOLVER')
diff -aurN trunk.orig/src/ladspa/wscript trunk/src/ladspa/wscript
--- trunk.orig/src/ladspa/wscript	2012-07-03 23:03:04.625627741 +0200
+++ trunk/src/ladspa/wscript	2012-07-03 23:22:07.144344289 +0200
@@ -42,7 +42,7 @@
     if sys.platform.startswith("linux"):
         lib.append('dl')
     uselib = ['JACK', 'SNDFILE', 'GTHREAD', 'GMODULE_EXPORT',
-              'GLIBMM', 'GIOMM', 'FFTW3']
+              'GLIBMM', 'GIOMM', 'FFTW3', 'BOOST_SYSTEM']
     if bld.env['ZITA_CONVOLVER']:
         uselib.append('ZITA_CONVOLVER')
     else:
diff -aurN trunk.orig/wscript trunk/wscript
--- trunk.orig/wscript	2012-07-03 23:03:04.617627841 +0200
+++ trunk/wscript	2012-07-03 23:03:40.936173794 +0200
@@ -346,7 +346,7 @@
     int main(){ return 0; }
     """ % boost_atleast_version
     msg = "Checking for boost >= %s" % boost_atleast_vermsg
-    conf.check_cxx(msg = msg, fragment=code, mandatory=1)
+    conf.check_cxx(msg = msg, fragment=code, lib="boost_system", mandatory=1)
     # not needed: with boost 1.42 Guitarix doesn't generate code which
     #             references the boost threading lib
     #msg = "Checking for boost_thread%%s >= %s" % boost_atleast_vermsg
